DroneState state
DroneControl control
std_msgs/Header header